### PR TITLE
Added mkdocs support

### PR DESCRIPTION
**Pull request by ar4s**
Connected to #1430

### CODE REVIEW

File: requirements/docs.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+mkdocs==0.12.2
```
Comment (mkurek) [inline requirements/docs.txt:1]: could you require (include) it in test.txt/dev.txt?

Comment (ar4s) [inline requirements/docs.txt:1]: I added `-r docs.txt` in `test.txt`

**Comment by ghost** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

**Pull request merged**